MatrixSDK/Utils/MXHTTPClient.m

```diff
@@ -81,6 +86,17 @@ -(id)initWithBaseURL:(NSString *)baseURL andAccessToken:(NSString *)access_token
         
         httpManager = [[AFHTTPRequestOperationManager alloc] initWithBaseURL:[NSURL URLWithString:baseURL]];
         
+        // Check whether some certificates have been included in app bundle
+        AFSecurityPolicy *securityPolicy = [AFSecurityPolicy policyWithPinningMode:AFSSLPinningModeCertificate];
+        if (securityPolicy.pinnedCertificates.count)
+        {
+            securityPolicy.allowInvalidCertificates = YES;
```

Comment:
If the certificate is included in app bundle, AFNetworking module will allow it without prompting the user. Is that your question?

It is not redundant, it is another way to handle certificate, if the developer uses its own certificates